src/cisco_gnmi/client.py

```diff
@@ -334,3 +334,19 @@ def parse_xpath_to_gnmi_path(self, xpath, origin=None):
             raise Exception("Unfinished elements in XPath parsing!")
         path.elem.extend(path_elems)
         return path
+
+    def parse_cli_command_to_gnmi_path(self, cli_command):
```

Comment:
I wasn't familiar with the ability to do CLI over gNMI with IOS XR - this explains the `ascii` support!
Is there any detail around what `origin` should be used in the path? Does it operate out of only the OpenConfig origin? Any origin?

Reply:
Tried to follow the gNMI specification but to keep the structure of the library when I defined the functions:

For reference, here is the specification regarding ASCII encoding https://github.com/openconfig/reference/blob/master/rpc/gnmi/gnmi-specification.md#234-ascii. One thing may be worth highlighting:

> The gNMI protocol does not define the format of the text – this must be established out-of-band.

There is no mention of origin outside the context of the xpath formatting.